### ISSUE TITLE
ActiveModel presence validation issue when using an enumerize with multiple: true 

### ISSUE THREAD

**Issue by BenjaminCbr** [brainspec/enumerize#247]
Hi,

I think there is a edge case in which Enumerize validation doesn't behave properly with ActiveModel built-in validations.
More specifically, the issue seems to occur when you provide a list containing only an empty string.

```ruby
class MyModel
  include ActiveModel::Model
  extend Enumerize

  enumerize :cities, in: %w(Boston London Berlin Paris), multiple: true
  validate :cities, presence: true
end


# Does not behave as expected
a = MyModel.new(cities: [""])
a.cities.present?  # => false
a.valid?  # => true, while it should be false.
```

Is there a workaround available to be able to use both Enumerize validation and ActiveModel presence validation?

It seems to work in some other cases (for instance when you provide a non empty string that is not in the Enum). For instance

```ruby
# Behaves as expected
a = MyModel.new(cities: ["Dublin"])
a.cities.present?  # => false
a.valid?  # => false
```

Thanks!

**Comment by nashby** [comment]
Looks like it's related: https://github.com/brainspec/enumerize/pull/226

**Comment by nashby** [comment]
fixed by #226 in master. @BenjaminCbr could you please test it in master branch?

**Comment by BenjaminCbr** [comment]
Yes the issue is indeed solve on master branch. Thanks for your help.
